### PR TITLE
DEVOPS-4808: Separate LMS and CMS doc base URLs

### PR DESCRIPTION
**Pull request by maxrothman**
FYI @catong @nasthagiri

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -286,7 +286,8 @@ EDXAPP_LANG: 'en_US.UTF-8'
 EDXAPP_LANGUAGE_CODE : 'en'
 EDXAPP_TIME_ZONE: 'America/New_York'
 
-EDXAPP_DOC_LINK_BASE_URL: 'http://edx.readthedocs.io/projects/open-edx-learner-guide'
+EDXAPP_LMS_DOC_LINK_BASE_URL: 'http://edx.readthedocs.io/projects/open-edx-learner-guide'
+EDXAPP_CMS_DOC_LINK_BASE_URL: 'http://edx.readthedocs.io/projects/open-edx-building-and-running-a-course'
```
Comment (nasthagiri) [inline playbooks/roles/edxapp/defaults/main.yml]: @catong Are these the correct links?
@maxrothman I believe they want different link values for Edge versus other environments.  Where would those be overridden?  In the config-secure repo?

Comment (maxrothman) [reply playbooks/roles/edxapp/defaults/main.yml]: These are the default values. The prod values are made in secure repos.

Comment (catong) [reply playbooks/roles/edxapp/defaults/main.yml]: @maxrothman These are the default URLs that are in the public configuration, for Open edX.
Where do we specify the "partner" flavors of the URLs that should be used on Edge and edx.org?

**Comment by maxrothman** [comment]
FWIW this is the diff in edge and prod:

```
-EDXAPP_DOC_LINK_BASE_URL: "http://edx.readthedocs.io/projects/edx-guide-for-students"
+EDXAPP_LMS_DOC_LINK_BASE_URL: "http://edx.readthedocs.io/projects/edx-guide-for-students"
+EDXAPP_CMS_DOC_LINK_BASE_URL: "http://edx.readthedocs.io/projects/edx-partner-course-staff"
```

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (maxrothman) [reply playbooks/roles/edxapp/defaults/main.yml]: [See my comment below](https://github.com/edx/configuration/pull/3391#issuecomment-248361887)

Comment (fredsmith) [reply playbooks/roles/edxapp/defaults/main.yml]: these should also be https

**Comment by catong** [comment]
Thanks @maxrothman, the new EDXAPP_LMS_DOC_LINK_BASE_URL and EDXAPP_CMS_DOC_LINK_BASE_URL values are what we want on edge and prod. 
Will we be able to test on stage?

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (nasthagiri) [reply playbooks/roles/edxapp/defaults/main.yml]: ah.  that's what I thought.  thanks for confirming.

**Comment by nasthagiri** [comment]
👍

**Review by fredsmith — APPROVED** [review]

**Comment by maxrothman** [comment]
@catong The changes apply to prod-edge, prod-edx, and stage-edx (and loadtest-edx FWIW).

File: playbooks/roles/edxapp/defaults/main.yml (hunk 1 above)
Comment (nasthagiri) [reply playbooks/roles/edxapp/defaults/main.yml]: and, they'll need the config-secure override on Stage as well.

**Comment by catong** [comment]
OK, thanks! 👍

**Pull request merged**